### PR TITLE
Add waiting room to hunting info

### PR DESCRIPTION
**Pull request by jayRyan24**
Could use testers for this!

### CODE REVIEW

**Comment by jayRyan24** [comment]
```
hunting_info:
- :zone:
  - void_moths
  - cabalist
  - archons
  args:
  - d0
  stop_on:
  - Heavy Thrown
  stop_on_low:
  - Locksmithing
  - Athletics
  full_waiting_room: ###
```

File: hunting-buddy.lic [hunk 1]
```diff
@@ -77,6 +77,7 @@ class HuntingBuddy
       stop_on_low_skills = info['stop_on_low']
       stop_on_boxes = info['boxes']
       stop_on_no_moons = info['moons']
+      waiting_room = @settings.safe_room || info['full_waiting_room']
```
Comment (rpherbig) [inline hunting-buddy.lic]: ❓ If I'm reading this right, `safe_room` will take precedence. I would have expected the more specific value, from the hunting info, to be higher priority. Thoughts?

**Comment by rpherbig** [comment]
✔️ Looks good aside from the one comment

File: hunting-buddy.lic (hunk 1 above)
Comment (jayRyan24) [reply hunting-buddy.lic]: Yes, I have it backwards!

**Comment by jayRyan24** [comment]
I'm good with this.

**Pull request merged**